src/doc/dev_manual/RCDevelopment.rst

```diff
@@ -83,6 +83,12 @@ and locally. ::
 Apply the same changes to develop
 ---------------------------------
 
+Typically, the same changes applied to the release candidate then also need to be applied to ``develop``.
```

Comment:
Maybe change "then also" to "also".

Reply:
> It looks to me like all RC PRs will now have comment at the end indicating it wasn't merged or it was merged with the PR number. Should this be stated explicitely somewhere?

Good idea. I will add in next commit

> Maybe change "then also" to "also".

Sure thing.